oio/conscience/checker/beanstalk.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2019 OpenIO SAS, as part of OpenIO SDS
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+from oio.event.beanstalk import Beanstalk
+from oio.conscience.checker.base import BaseChecker
+
+
+class BeanstalkChecker(BaseChecker):
+    name = 'beanstalkd'
+
+    def configure(self):
+        print("XXX")
+        for k in ['host', 'port', 'tube']:
+            if k not in self.checker_conf:
+                raise exc.ConfigurationException(
+                    'Missing field "%s" in configuration' % k)
+        print("YYY")
+
+    def check(self):
+        result = False
+        cnx = None
+        try:
+            cnx = Beanstalk(self.checker_conf['host'], self.checker_conf['port'])
+            cnx.stats_tube(self.checker_conf['tube'])
```

Comment:
I think we only need to check the state of the service, not a specific tube. Calling `stats` would be better (must be implemented in `beanstalk.py`).